cluster/proto/api/message.proto

```diff
@@ -2,12 +2,26 @@ syntax = "proto3";
 
 package weaviate.internal.cluster;
 
+// protoc \
+// --go_out=paths=source_relative:. \
+// --go-grpc_out=paths=source_relative:. \
+// --go-grpc_opt=require_unimplemented_servers=false \
+// api/*.proto
+
+// TODO can i use "--go_opt=M" to avoid needing this?
+// option go_package = "github.com/weaviate/weaviate/cluster/proto/api";
+
 service ClusterService {
   rpc RemovePeer(RemovePeerRequest) returns (RemovePeerResponse) {}
   rpc JoinPeer(JoinPeerRequest) returns (JoinPeerResponse) {}
   rpc NotifyPeer(NotifyPeerRequest) returns (NotifyPeerResponse) {}
   rpc Apply(ApplyRequest) returns (ApplyResponse) {}
   rpc Query(QueryRequest) returns (QueryResponse) {}
+
+  // QuerierStream is experimental, may be changed/removed. A QuerierStream represents a
+  // connection between this metadata cluster and a querier node. See the implementing
+  // function's doc for more information.
+  rpc QuerierStream(stream QuerierStreamRequest) returns (stream QuerierStreamResponse) {}
```

Comment:
Should we move this out into a separate service? I think we'll likely modify/remove this rpc multiple times as we iterate.

Reply:
I'd be in favor of keeping it out in a different service (maybe we make it listen on a different port as well on the weaviate nodes ? Can we have 2 service listening on the port and reuse the raft internal port ?)

Also it's a different area of responsibility `ClusterService` is related to metadata read/write and raft cluster mgmt.

@moogacs what do you think ?

Reply:
I agree, I'll look into creating a neew grpc service/port